Shared/Prefs.swift

```diff
@@ -127,7 +127,7 @@ public struct PrefsKeys {
     public static let WallpaperManagerLogoSwitchPreference = "WallpaperManagerLogoSwitchPreference"
 
     // Application Services Migration to Places DB
-    public static let NewPlacesAPIDefaultKey = "NewPlacesAPI"
+    public static let HistoryMigratedToPlacesKey = "PlacesHistoryMigrationRanreal"
```

Comment:
I'm not sure what `Ran real` means ? 
```suggestion
    public static let HistoryMigratedToPlacesKey = "PlacesHistoryMigrationRanReal"
```

Reply:
Yeah the context here is that the `real` at the end used to come from the nimbus configuration - i.e if a user was migrated due to a nimbus rollout (which I'm evaluating based on the experiment) they don't get migrated again when they upgrade to 110